aries-site/src/pages/components/page.mdx

```diff
@@ -0,0 +1,170 @@
+import { Example } from '../../layouts';
+import { 
+  PageBackGroundExample,
+  PageExample, 
+  PageFullExample,
+  PageNarrowExample,
+  SinglePageContent,
+  PageWideExample 
+} from '../../examples';
+
+Page makes it easy to implement [page layouts](/templates/page-layouts) uniformly across HPE 
+applications. Two components, Page and PageContent, work together to ensure HPE theme-defined 
+padding, horizontal alignment, max-width, and responsive behaviors are applied in a consistent, 
+turn-key manner.
+
+<Example 
+  docs="https://v2.grommet.io/page?theme=hpe"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js"
+  showResponsiveControls={['fullScreen']}
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  screenContainer={{scale: 0.85}}
+  width="100%"
+>
+  <PageExample />
+</Example>
+
+## Page and PageContent Working Together
+Page is the outer layout component and provides the desired layout context. 
+Page’s `kind` determines which theme-defined alignment, padding, and min/max-widths 
+are applied to its PageContent children.
+
+As a child of Page, PageContent consumes and adheres to the 
+layout styling provided by its parent Page.
+
+```
+<Page kind="wide">
+  <PageContent>
+
+    /* All contents within me will respect the 
+     * max-width, alignment, and padding specified 
+     * by Page's context. */
+     
+  </PageContent>
+  
+  /* Contents outside of PageContent are not bound by 
+   * Page's context. */
+
+</Page>
+```
+
+<br />
+
+In most cases, all content on a page can be wrapped in a [single PageContent component](#simple). 
+However, designs using [differing background colors](#full-width-background-color) to delineate page 
+sections may accomplish this by applying a PageContent per desired section.
+
+## Example use cases
+
+### Simple
+All content is placed within a single PageContent component.
+
+<Example
+  code={[
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/page/SinglePageContent.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/FilterControls/FilterControls.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/FilterControls/FiltersContext.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/PageHeader/PageHeader.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/data/mockData/trainingDataSets.json",
+  ]}
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  width="100%" 
+  showResponsiveControls={['fullScreen']}
+  screenContainer={{ scale: 0.6 }}
+>
+  <SinglePageContent />
```

Comment:
I'm finding the amount of dashed boxes confusing. It feels like the dashed boxes should be used to highlight the components specific to this page (Page and PageContent), so I would only expect to have 2 dashed boxes.

The dashed boxes relating to how content is built up seems too tangential to the topic.

<img width="1536" alt="Screen Shot 2022-04-05 at 2 47 04 PM" src="https://user-images.githubusercontent.com/12522275/161855689-e7646dc8-36e3-490e-991a-d309499f02ed.png">

Reply:
I agree. Presented option for reaction. Will only apply to Page and PageContent.

Reply:
I also agree I liked the dashed line on the first example made it easier to follow

Reply:
done